pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -121,3 +121,41 @@ Examples:
 	cmd.Long = fmt.Sprintf(longDesc, fullName)
 	return cmd
 }
+
+func NewCmdExec(fullName string, f *clientcmd.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer) *cobra.Command {
+	cmd := f.NewCmdExec(cmdIn, cmdOut, cmdErr)
+	longDesc := `Execute a command in a container.
+
+Examples:
+
+	# get output from running 'date' in ruby-container from pod 123456-7890
+	$ %[1]s exec -p 123456-7890 -c ruby-container date
+
+	# switch to raw terminal mode, sends stdin to 'bash' in ruby-container from pod 123456-780 and sends stdout/stderr from 'bash' back to the client
+	$ %[1]s exec -p 123456-7890 -c ruby-container -i -t -- bash -il
+`
+	cmd.Long = fmt.Sprintf(longDesc, fullName)
+	return cmd
+}
+
+func NewCmdPortForward(fullName string, f *clientcmd.Factory) *cobra.Command {
+	cmd := f.NewCmdPortForward()
+	longDesc := `Forward 1 or more local ports to a pod.
+
+Examples:
+
+	$ %[1]s port-forward -p mypod 5000 6000
+	<listens on ports 5000 and 6000 locally, forwarding data to/from ports 5000 and 6000 in the pod>
```

Comment:
If you don't mind, try keeping these examples consistent with the rest of the examples.